Compilation error about vxid

Hello,

Trying to compile for PostgreSQL V15 or V16 on RHEL7 or RHEL8, we get this error:

```
$ USE_PGXS=1 make
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wimplicit-fallthrough=3 -Wcast-function-type -Wshadow=compatible-local -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -Wno-stringop-truncation -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fPIC -fvisibility=hidden -I. -I./ -I/usr/pgsql-16/include/server -I/usr/pgsql-16/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include  -c -o snowflake.o snowflake.c
snowflake.c: In function ‘lock_and_open_sequence’:
snowflake.c:459:40: error: ‘PGPROC’ {aka ‘struct PGPROC’} has no member named ‘vxid’; did you mean ‘xid’?
  LocalTransactionId thislxid = MyProc->vxid.lxid;
                                        ^~~~
                                        xid
make: *** [<builtin>: snowflake.o] Error 1
```

--- a/snowflake.c
+++ b/snowflake.c
@@ -456,7 +456,11 @@ snowflake_get_node(PG_FUNCTION_ARGS)
 static Relation
 lock_and_open_sequence(SeqTable seq)
 {
+#if PG_VERSION_NUM >= 170000
 	LocalTransactionId thislxid = MyProc->vxid.lxid;
+#else
+	LocalTransactionId thislxid = MyProc->lxid;
+#endif
 
 	/* Get the lock if not already held in this xact */
 	if (seq->lxid != thislxid)

